src/python/pants/bin/pants_loader.py

```diff
@@ -33,6 +33,9 @@ def setup_warnings():
     # TODO: Eric-Arellano has emailed the author to see if he is willing to accept a PR fixing the
     # deprecation warnings and to release the fix. If he says yes, remove this once fixed.
     warnings.filterwarnings('ignore', category=DeprecationWarning, module="ansicolors")
+    # Silence this ubiquitous warning. Several of our 3rd party deps incur this.
+    warnings.filterwarnings('ignore', category=DeprecationWarning,
+        message="Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated")
```

Comment:
I agree with muting this warning in principle because it has been verbose, and the reason in the commit msg makes sense. Writing the full message looks potentially fragile though as that feels like it may change faster than the actual warning. Is there some kind of warning category that we can use to describe this warning in a more generic way?

Reply:
There is no other way to identify this specific message. If the message changes, we can update here. I see no reason it should change.

Reply:
Agreed that there is no reason for this to change.